### DISCUSSION TITLE
is there a way to choose a controlnet pose from a folder of poses - 1 at a time at random - instead of run the whole batch?

### DISCUSSION THREAD

**Discussion by Sacred-Reputation** [lllyasviel/ControlNet#486] [General]
i have a folder of canny poses - i'd like it to choose 1 at random each time - i can get it to run the whole batch, but it always does them in order of filename. anyway to have controlnet to randomize the batch each time?

**Comment by geroldmeisinger** [comment]
this is a UI specific question, but yes, you can do something like that in Automatic111 ControlNet -> Batch
